group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupConfigManager.java

```diff
@@ -67,6 +67,13 @@ public Optional<GroupConfig> groupConfig(String groupId) {
         return Optional.ofNullable(configMap.get(groupId));
     }
 
+    public Optional<Integer> getShareGroupRecordLockDurationMs(String groupId) {
+        if (!groupConfig(groupId).isPresent()) {
```

Comment:
Also, the optionality here is only partially correct. It there is no config set for the group, it returns empty. But, if I have set any configs for the group, regardless of whether it was the share group record lock duration, a value will be returned.